lib/metadata/wrapper.js

```diff
@@ -67,54 +67,25 @@ const metadata = {
         });
     },
 
-    putObjectMD: (bucketName, objName, objVal, log, cb, params) => {
+    putObjectMD: (bucketName, objName, objVal, params, log, cb) => {
         log.debug('putting object in metdata');
-        client.putObject(bucketName, objName, objVal, log, err => {
-            if (err) {
```

Comment:
We're downgrading the warn log level to debug so there won't be confusing error logs in production at this level.  

I agree with @electrachong and think we should be mindful of what arguments we are getting back from bucketclient metadata (and the other metadata backends should provide the same arguments) so updating the arguments here would be good so we are clear on what is expected from any metadata backend.